typings/love.graphics/functions.d.ts

```diff
@@ -1307,7 +1307,7 @@ declare module "love.graphics" {
      * @link [love.graphics.newShader](https://love2d.org/wiki/love.graphics.newShader)
      * @since 0.9.0
      */
-    function newShader<T = undefined>(code: string): Shader<T>;
+    function newShader<T extends {[key:string]:any} = {}>(code: string): Shader<T>;
```

Comment:
```suggestion
    function newShader<T extends {[key:string]:any} = { [key: string]: any }>(code: string): Shader<T>;
```

A default constraint of `{ [key: string]: any }` will allow users to continue to use the following code:

```ts
const shader = love.graphics.newShader("...");
shader.send("myUniform", 0);
```

If you could apply this change to line 1330 and run `yarn fix:prettier` I'm happy to get this merged and released